guide/v0/forms.md

```diff
@@ -0,0 +1,83 @@
+# Forms
+
+The PTI Forms service exposes the various forms used to collect information on users.
+The SDK tool uses this service to provide the different forms. Form customization is available through scenarios
+and client configuration. 
+
+
+## Client configuration
+
+The client configuration can be used to customize form styles, translations and more.
+The config should follow this format:
+
+```typescript
+interface ClientConfig extends Entity {
+  acceptedCc: string[];
+  clientId: string;
+  feeRate: number;
+  exceptionLists: ExceptionLists;
+  styleConfig: StyleConfig;
+  translations: TranslationConfig;
+}
+```
+
+- `acceptedCc` is a list of allowed credit card providers. You can restrict usage of certain credit card providers 
+by allowing a subset of the default value `["amex", "dinersclub", "discover", "jcb", "mastercard", "visa"]`
+- `clientId` is your assigned client id
+- `feeRate` is the desired fee rate when processing payments. This should be the decimal value of the
+added fee, for instance `0.03` for a 3% fee.
+
+### Exception Lists
+
+It is also possible to customize the restriction of certain areas, or flag high risk ones. 
+The `exceptionLists` config should follow the following format: 
+
+```typescript
+interface ExceptionLists {
+  embargoedList: string[];
+  highRiskList: string[];
+  regulatoryIssuesList: string[];
+  usStateBlacklist: string[];
+}
+```
+
+- `embargoedList` is a list of country codes where service providing should be denied. A default list will be provided
```

Comment:
If list is short provide it, else point to it in the code.

Reply:
List isn't short, but since Form's repo is private I'll provide it here. Thanks!